modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -86,7 +86,9 @@ def json_schemer_error_to_string(error)
 
     FORM_SCHEMA = load_json_schema '10182'
     AUTH_HEADERS_SCHEMA = load_json_schema '10182_headers'
+    STATUSES = %w[pending].freeze
 
+    validates :status, inclusion: { 'in': STATUSES }
```

Comment:
We'll need to think about how we want to convert rails validations into Json API ErrorObject notation 🤔 Not a blocker, just something to ponder.